scripts/dev_setup.py

```diff
@@ -25,13 +25,15 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install general requirements
+exec_command('pip install -r requirements.txt')
```

Comment:
@tjprescott There was a reason we has the install of requirements last.
It was to prevent command modules that are installed later from overriding what we have in requirements.txt.
It was with `azure-mgmt-keyvault` back when we were using the `azure` rollup package.

@troydai Is this particular change required for this fix to work?